packages/graphiql-plugin-react-doc-explorer/.eslintrc

```diff
@@ -0,0 +1,22 @@
+{
+  "extends": [
```

Comment:
```suggestion
  "root": true,
  "extends": [
```
this config do nothing, without `root: true` always root eslint config is taken in priority